src/riak_test_escript.erl

```diff
@@ -325,7 +325,16 @@ run_test(Test, TestType, Outdir, TestMetaData, Report, HarnessArgs, NumTests) ->
                     [giddyup:post_artifact(Base, {filename:basename(CoverageFile) ++ ".gz",
                                                   zlib:gzip(element(2,file:read_file(CoverageFile)))}) || CoverageFile /= cover_disabled ],
                     ResultPlusGiddyUp = TestResult ++ [{giddyup_url, list_to_binary(Base)}],
-                    [ rt:post_result(ResultPlusGiddyUp, WebHook) || WebHook <- get_webhooks() ]
+                    [ rt:post_result(ResultPlusGiddyUp, WebHook) || WebHook <- get_webhooks() ],
+                    CTLogTarFile = "/tmp/ctlogs_" ++ integer_to_list(erlang:phash2(make_ref())),
+                    case erl_tar:create(CTLogTarFile, ["ct_logs"], [compressed]) of
+                        ok ->
+                            giddyup:post_artifact(Base, {"ct_logs.tar.gz",
+                                element(2,file:read_file(CTLogTarFile))});
```

Comment:
Can you do a `{ok, Contents} = file:read_file(CTLogTarFile)` instead of using `element/2`?